assets/animations.js

```diff
@@ -22,7 +22,13 @@ function initializeScrollAnimationTrigger(rootEl = document) {
   const observer = new IntersectionObserver(onIntersection, {
     threshold: 0.1,
   });
-  animationTriggerElements.forEach((element) => observer.observe(element));
+  animationTriggerElements.forEach((element) => {
+    if (element.hasAttribute('data-3d-model')) {
```

Comment:
In products where the media gallery has a 3D model as its first image, we're seeing an issue where the 3D model and product info don't fade/slide in until you start scrolling. This is the only solution I could think of, as this seems to be caused by the threshold here, but I'm open to suggestions if anyone has a better solution.

https://user-images.githubusercontent.com/22390758/229139039-ad718ebf-f1f1-4e5a-bc9f-83e2f603fa50.mp4

Reply:
I think the issue is just that, for this product, the elements are super tall! I see them having a height of almost 7000 pixels. So, with a threshold of 0.1 it's actually behaving correctly by waiting until you scroll down a bit.

If you put the `scroll-trigger` on the product media items instead it might work better.

Reply:
Are there specific steps to reproduce this issue on the latest main? I just removed this `data-3d-model` and the JS block above and it seems to be working as expected on the same product as shown in the video (`A/ Image sizing test – 3D models and video`) for both Product Info and Featured Product sections, both inside and outside the Editor.

So I wonder if this was fixed by the **[fade in each image](https://github.com/Shopify/dawn/pull/2405/commits/7b00c16e26cd2c731a055bba1d0a78a9f4f751f4)** change you made or if I'm missing something to reproduce it.

Reply:
Are there specific steps to reproduce this issue on the latest status of this branch?

I just removed this `data-3d-model` and the JS block above and it seems to be working as expected on the same product as shown in the video (`A/ Image sizing test – 3D models and video`) for both **Product Info** and **Featured Product** sections, both inside and outside the Editor.

So I wonder if this was fixed by the **[fade in each image](https://github.com/Shopify/dawn/pull/2405/commits/7b00c16e26cd2c731a055bba1d0a78a9f4f751f4)** change you made or if I'm missing something to reproduce it.

Reply:
Are there specific steps to reproduce this issue on the latest status of this branch?

I just removed this `data-3d-model` and the JS block above and it seems to be working as expected on the same product as shown in the video (`A/ Image sizing test – 3D models and video`) for both **Product Info** and **Featured Product** sections, both inside and outside the Editor.

So I wonder if this was fixed by the **[fade in each image](https://github.com/Shopify/dawn/pull/2405/commits/7b00c16e26cd2c731a055bba1d0a78a9f4f751f4)** change you made or if I'm missing something to reproduce it.

Reply:
The issue with the 3D model was fixed by a combination of @stufreen's changes and changing the animation from slide-in to fade-in, so this should work now.